src/notifiers/pagerduty_notifier.py

```diff
@@ -0,0 +1,73 @@
+from logging import getLogger
+from typing import Any, Dict, List, Set
+
+import requests
+from src.config.config import Config
+from src.data.exceptions import PagerDutyNotifierException
+from src.data.pagerduty_event import PagerDutyEvent
+from src.data.pagerduty_payload import PagerDutyPayload
+
+from src.notifiers.notifier import Notifier
+from src.pagerduty_notification_mapper import PagerDutyNotificationMapper
+from src.pagerduty_payload_filter import PagerDutyPayloadFilter
+
+# moved to pagerduty_notification_mapper.py
```

Comment:
What has moved there?

Reply:
It was to remind me to clean up temp variables. Deleted now as they are not used in the module.